hazelcast-sql/src/test/java/com/hazelcast/sql/impl/expression/math/AbsFunctionIntegrationTest.java

```diff
@@ -0,0 +1,295 @@
+
+@SuppressWarnings({"rawtypes", "unchecked"})
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelJVMTest.class})
+public class AbsFunctionIntegrationTest extends SqlTestSupport {
+
+    private final TestHazelcastInstanceFactory factory = new TestHazelcastInstanceFactory(1);
+    private HazelcastInstance member;
+    private IMap map;
+
+    @Before
+    public void before() {
+        member = factory.newHazelcastInstance();
+
+        map = member.getMap("map");
+    }
+
+    @After
+    public void after() {
+        factory.shutdownAll();
+    }
+
+    @Test
+    public void testColumn() {
+        checkColumn((byte) 0, SqlColumnType.SMALLINT, (short) 0);
+        checkColumn((byte) 1, SqlColumnType.SMALLINT, (short) 1);
+        checkColumn((byte) -1, SqlColumnType.SMALLINT, (short) 1);
+        checkColumn(Byte.MAX_VALUE, SqlColumnType.SMALLINT, (short) Byte.MAX_VALUE);
+        checkColumn(Byte.MIN_VALUE, SqlColumnType.SMALLINT, (short) (Byte.MAX_VALUE + 1));
+
+        checkColumn((short) 0, SqlColumnType.INTEGER, 0);
+        checkColumn((short) 1, SqlColumnType.INTEGER, 1);
+        checkColumn((short) -1, SqlColumnType.INTEGER, 1);
+        checkColumn(Short.MAX_VALUE, SqlColumnType.INTEGER, (int) Short.MAX_VALUE);
+        checkColumn(Short.MIN_VALUE, SqlColumnType.INTEGER, Short.MAX_VALUE + 1);
+
+        checkColumn(0, SqlColumnType.BIGINT, (long) 0);
+        checkColumn(1, SqlColumnType.BIGINT, (long) 1);
+        checkColumn(-1, SqlColumnType.BIGINT, (long) 1);
+        checkColumn(Integer.MAX_VALUE, SqlColumnType.BIGINT, (long) Integer.MAX_VALUE);
+        checkColumn(Integer.MIN_VALUE, SqlColumnType.BIGINT, (long) Integer.MAX_VALUE + 1);
+
+        checkColumn((long) 0, SqlColumnType.BIGINT, (long) 0);
+        checkColumn((long) 1, SqlColumnType.BIGINT, (long) 1);
+        checkColumn((long) -1, SqlColumnType.BIGINT, (long) 1);
+        checkColumn(Long.MAX_VALUE, SqlColumnType.BIGINT, Long.MAX_VALUE);
+        checkColumnFailure(Long.MIN_VALUE, SqlErrorCode.DATA_EXCEPTION, "BIGINT overflow in ABS function (consider adding an explicit CAST to DECIMAL)");
+
+        checkColumn(BigInteger.ZERO, SqlColumnType.DECIMAL, BigDecimal.ZERO);
+        checkColumn(BigInteger.ONE, SqlColumnType.DECIMAL, BigDecimal.ONE);
+        checkColumn(BigInteger.ONE.negate(), SqlColumnType.DECIMAL, BigDecimal.ONE);
+
+        checkColumn(BigDecimal.ZERO, SqlColumnType.DECIMAL, BigDecimal.ZERO);
+        checkColumn(BigDecimal.ONE, SqlColumnType.DECIMAL, BigDecimal.ONE);
+        checkColumn(BigDecimal.ONE.negate(), SqlColumnType.DECIMAL, BigDecimal.ONE);
+
+        checkColumn((float) 0, SqlColumnType.REAL, (float) 0);
+        checkColumn((float) 1.1, SqlColumnType.REAL, (float) 1.1);
+        checkColumn((float) -1.1, SqlColumnType.REAL, (float) 1.1);
+        checkColumn(Float.MAX_VALUE, SqlColumnType.REAL, Float.MAX_VALUE);
+        checkColumn(Float.MIN_VALUE, SqlColumnType.REAL, Math.abs(Float.MIN_VALUE));
```

Comment:
`Float.MIN_VALUE` is a positive number actually...

Reply:
Removed these tests